who-metrics-ui/src/components/RepositoriesTable.tsx

```diff
@@ -224,6 +225,26 @@ const defaultFilters: Filter = {
   },
 };
 
+// Helper for generating the csv blob
+const generateCSV = (data: Repo[]): Blob => {
```

Comment:
[nit] We might want to put these helpers in a different file and import them in, just for readability?

Reply:
I was thinking about breaking up this whole file entirely since it's getting pretty large. I can start that here in this PR or address in a follow up